code/src/Components/Album.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import Buttons from './Buttons.js'
+
+//image needs an URL
+const Album = (props) => {
+  console.log(props)
+    return (
+      <div className="album-card">
+          <div className="image-container">            
+              <img className="album-image" src={props.imgsrc}></img>          
+            <Buttons />
+          </div>        
+        <div className="album-information-container">
+          <a className="url-link" href={props.url} target="_blank">
+            <h3 className="album-name">{props.name}</h3>
+          </a>
+        </div>
+
+        {props.artist.map((artist) => {
+          console.log(artist)  
```

Comment:
remember to remove console.logs  from your code after you finish debugging your code.